### PR TITLE
Avocado and Cucumber plants, with foods

### PR DESCRIPTION
**Pull request by dustylens**
This PR adds avocado, cucumber, and plants for each.

In addition it adds several food items that utilize one of these new produces or both. 

## About the PR

Two new plants with seeds and growth stages.
Three new sandwiches.
Two new sushi types.
Avocado toast.
Two salads.
A refreshing beverage.

## Why / Balance

Avocado has been in the holding pattern for a while. The time is avocaNOW.
Many recipes that made use of avocado, such as sushi, felt incomplete without cucumber. So I went a bit further than I needed. 

Cucumber nutritional value should be less than its peers but is balanced out with the cucumber water reagent, which has the same thirst satiation as saline. So a nice little refreshing boost.

## How to test

Purchase avocado and cucumber seeds. Attempt to grow. Harvest. Try to reclaim seeds. Attempt to kill. 

Attempt to assemble included food meals. Note that produce also has pickle. Which is made by combining saline and cucumber. It's a good placeholder for now and a helpful food ingredient for later.

## Media

toofar

## Requirements

- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

Hope not.

**Changelog**

:cl:
-add: Adds avocado and cucumber plants in addition to new supported recipes and reagents.

### CODE REVIEW

File: Resources/Prototypes/_NF/Entities/Objects/Consumable/Food/meals.yml [hunk 1]
```diff
@@ -96,3 +96,182 @@
...
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 9
+        - ReagentId: Vitamin
+          Quantity: 5
+        - ReagentId: CucumberWater
+          Quantity: 10 
+  - type: Tag
+    tags:
+    - Meat
```
Comment (ErhardSteinhauer) [inline Resources/Prototypes/_NF/Entities/Objects/Consumable/Food/meals.yml:211]: Meat?

**Review by arimah — COMMENTED** [review]
The seed packet sprites seem to be based on the old style from nearly a year ago. You may wish to update to the new style established by this upstream PR that _almost_ all seed packets on Frontier now use: https://github.com/space-wizards/space-station-14/pull/23210

**Comment by whatston3** [comment]
Suggestions up at [whatston3:Bingacado-suggestions](https://github.com/DustyLens/frontier-station-14/compare/Bingacado...whatston3:frontier-station-14:Bingacado-suggestions?expand=1) - mostly sprite touch-ups and some whitespace cleanup.  Changed a few reagents in some recipes to be properly proportional.  Removed the carrot from the carp sushi (unsure about that one but seems odd to me, thought it was more california roll), and reduced the ingredients in one of the salads to be more consistent with the other.  Sushi seems a little strong but it's still better than citrus salad.  Good feature overall.

Made the cucumber greens a bit lighter and less saturated, fiddled with the pickle colours, lengthened the cucumber, and made the right side of the avocado a bit darker and changed the hue slightly towards green.  The sandwich was not changed, the newer greens looked worse against the orange/brown bread.

Do give the sprites a look, I don't have a great eye for these things.

Nice little feature.

**Comment by whatston3** [comment]
Also, for what it's worth, I do think getting regular oil out of the avocado is weird.  Even if it wasn't used for anything else other than fryer oil, I think a separate reagent's a reasonable addition.

**Comment by whatston3** [comment]
Did add a vegetable oil reagent (no more oils please), made it fryable, swapped out base oil for OilVegetable.  It's up in that suggestions branch, last commit, omit it if it's excessive.

**Comment by arimah** [comment]
I'm completely in favour of adding OilVegetable to the game, and if we do, we should 100% also update the mayonnaise recipes to use it :D

**Review by whatston3 — APPROVED** [review]

File: Resources/Prototypes/_NF/Entities/Objects/Consumable/Food/produce.yml [hunk 2]
```diff
@@ -64,3 +64,205 @@
...
+  - type: Produce
+    seedId: avocado
+  - type: SliceableFood
+    count: 2
+    slice: FoodAvocadoSlice
+  - type: Extractable
+    grindableSolutionName: food
+    juiceSolution:
+      reagents:
+      - ReagentId: OilVegetable
```
Comment (ErhardSteinhauer) [inline Resources/Prototypes/_NF/Entities/Objects/Consumable/Food/produce.yml:97]: Is this supposed to be `Oil` or `OilVegetable`?

File: Resources/Prototypes/_NF/Entities/Objects/Consumable/Food/produce.yml [hunk 3]
```diff
@@ -64,3 +64,205 @@
...
+          Quantity: 5
+        - ReagentId: Vitamin
+          Quantity: 3
+  - type: Sprite
+    sprite: _NF/Objects/Specific/Hydroponics/avocado.rsi
+  - type: Extractable
+    grindableSolutionName: food
+    juiceSolution:
+      reagents:
+      - ReagentId: Oil
```
Comment (ErhardSteinhauer) [inline Resources/Prototypes/_NF/Entities/Objects/Consumable/Food/produce.yml:127]: Is this supposed to be `Oil` or `OilVegetable`?

**Review by ErhardSteinhauer — DISMISSED** [review]

File: Resources/Prototypes/_NF/Entities/Objects/Consumable/Food/produce.yml (hunk 3 above)
Comment (whatston3) [reply Resources/Prototypes/_NF/Entities/Objects/Consumable/Food/produce.yml]: Yeah, I messed up.  Should be pushed.

**Review by ErhardSteinhauer — APPROVED** [review]

**Pull request merged**